src/index.tsx

```diff
@@ -245,6 +246,10 @@ export default class AppIntroSlider<ItemT = any> extends React.Component<
   };
 
   _onMomentumScrollEnd = (e: {nativeEvent: NativeScrollEvent}) => {
+    this._onScrollEnd(e);
+  };
```

Comment:
Looks like this function now serves no purpose, I'd remove it and just set `onMomentumScrollEnd={this._onScrollEnd}` in the render